internal/parser/tmcf.go

```diff
@@ -93,6 +93,7 @@ func ParseTmcf(tmcf string) (map[string]*TableSchema, error) {
 			for _, prefix := range []string{"dcs:", "dcid:", "schema:"} {
 				schema = strings.TrimPrefix(schema, prefix)
 			}
+			schema = strings.Trim(schema, "\"")
```

Comment:
comment? Is it to avoid enclosing quotes in string values, or something else?

Reply:
Yes, it's for enclosing quotes like "P1M" in TMCF, added comment